flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -47,5 +47,13 @@ EXIT_CODE=0
 #     EXIT_CODE=$?
 # fi
 
+if [ $EXIT_CODE == 0 ]; then
+    printf "\n===============================================================================\n"
+    printf "Running Elasticsearch1 nightly end-to-end test\n"
```

Comment:
I would place this test after the Kafka end-to-end tests.

Reply:
Why ?

Reply:
Oh. I see. Sorry for misunderstanding. I will put it after kafka.